src/api/localStorage.js

```diff
@@ -0,0 +1,93 @@
+export class LocalStorage {
+  constructor() {
+    this.users = JSON.parse(localStorage.getItem('userAccounts'));
+  }
+
+  createUser(user) {
+    if (this.users === null) this.users = [];
+    user._id = Math.floor(10000 + Math.random() * 9000);
+    this.users.push(user);
+    localStorage.setItem('UserAccounts',
```

Comment:
you'll have to make sure you're `getItem` and `setItem` naming is consistent. I would think with the way you've got it set up now, you're always setting `UserAccounts` but getting from `userAccounts`